modules/configure-cvo-retrieve-image-signatures.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configure-cvo-retrieve-image-signatures_{context}"]
+= Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+
+If you are using the oc-mirror plugin 4.15 or later and the OSUS Operator 5.0.3 or later to update your cluster in a disconnected environment, you can configure the Cluster Version Operator (CVO) to retrieve release image signatures from an endpoint on the OpenShift Update Service (OSUS) in your environment.
+
+This configuration is done by adding the URI of OSUS' signature serving endpoint to the CVO's list of signature stores through the `signatureStores` property.
+This is a one-time configuration that avoids the need to apply release image signatures each time you mirror a new set of release images.
+
+:FeatureName: The `signatureStores` property
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have deployed OSUS using the OSUS Operator 5.0.3 or later.
+* You are using the oc mirror plugin 4.15 or later to mirror release images.
+
+.Procedure
+
+. Retrieve the signature serving URI of OSUS by running the following command:
++
+[source,terminal]
+----
+$ while sleep 1; do SIGNATURES_URI=“$(oc -n “${NAMESPACE}” get -o jsonpath=‘{.status.metadataURI}/api/upgrades_info/signatures/{“\n”}’ updateservice “${NAME}“)”; SCHEME=“${SIGNATURES_URI%%:*}“; if test “${SCHEME}” = http -o “${SCHEME}” = https; then break; fi; done
```

Comment:
It seems `“` should be `"`, `‘` should be `'`?

Reply:
I ran below command on my cluster which has OSUS and updateservice installed, but stucked:
```
[jianl@localhost 415]$ NAME=sample
[jianl@localhost 415]$ NAMESPACE=openshift-update-service
[jianl@localhost 415]$ while sleep 1; do SIGNATURES_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.metadataURI}/api/upgrades_info/signatures/{"\n"}' updateservice "${NAME}")"; SCHEME="${SIGNATURES_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done


```

Reply:
@JianLi-RH I can fix the quotation marks, sure. 

When you ran the command, did you directly copy and paste from this line, or did you change the quotation marks yourself before running the command?

Reply:
I copy the command line from UI to notebook, then update quotations, then run it.

Copy it to terminal doesn't work.

Reply:
@JianLi-RH understood, thanks for explaining.

@jiajliu I'll remove the `/` from that part of the path, thanks.